sql/mysql/cdash.sql

```diff
@@ -548,6 +548,7 @@ CREATE TABLE `build2test` (
   `timestd` float(7,2) NOT NULL default '0.00',
   `timestatus` tinyint(4) NOT NULL default '0',
   `newstatus` tinyint(4) NOT NULL default '0',
+  PRIMARY KEY (`buildid`, `testid`),
```

Comment:
Nervous about the constraint that each build can only perform each test once.  Will think this over a bit more and report back.

Reply:
Makes sense.  As I said this gave us almost no improvements.